### DISCUSSION TITLE
Error Occurred : '异常'

### DISCUSSION THREAD

**Discussion by luckyyangrun** [PaddlePaddle/PaddleSpatial#102] [BaiduKDDCup2022] [answered]
Dear Organizer,
I got an error ' Error Occurred : '异常' ' My codebase can pass the previous evaluations and local test. What is the meaning of this error?
![image](https://user-images.githubusercontent.com/67430592/170194998-f5a6757e-848d-4300-8715-bf05aafdb9ef.png)

**Comment by seeingisbelieveing** [comment]
Dear participant, could you please provide your team name? We will check it, Thx.

**Reply by luckyyangrun** [reply]
Thank you !! My team name is 'sufe-tj'

**Reply by QiZhou1997** [reply]
Dear organizer, I also get stuck with a similar issue. My code can pass previous evaluations and local test. However, I got the error message " ‘Nonetype’ object has no attribute ‘forcast’ “ today.  My team name is ‘Up to 20 characters’. Could you please check the error logs? Thank you!

**Reply by seeingisbelieveing** [reply]
> 

Please also include your team name and submit time, thanks!

**Reply by QiZhou1997** [reply]
> 
Thank you for quick reply. My team name is ‘Up to 20 characters’. We get the first error message in 2022-05-25 19:57.
<img width="619" alt="image" src="https://user-images.githubusercontent.com/26005993/170289194-cf962805-c004-4c89-8cd5-c88f904d8a88.png">
I have tried submitting code several times after that. The last one is in 2022-05-25 22:27.
<img width="461" alt="image" src="https://user-images.githubusercontent.com/26005993/170289251-4689493e-0fa1-4f5e-82b3-24f0d01eb87a.png">

**Reply by seeingisbelieveing** [reply]
> > 
> 
> Thank you for quick reply. My team name is ‘Up to 20 characters’. We get the first error message in 2022-05-25 19:57. <img alt="image" width="619" src="https://user-images.githubusercontent.com/26005993/170289194-cf962805-c004-4c89-8cd5-c88f904d8a88.png"> I have tried submitting code several times after that. The last one is in 2022-05-25 22:27. <img alt="image" width="461" src="https://user-images.githubusercontent.com/26005993/170289251-4689493e-0fa1-4f5e-82b3-24f0d01eb87a.png">

The log displayed "ModuleNotFoundError: No module named 'tqdm'". Did you put 'tqdm' in your pip requirement file?

**Reply by seeingisbelieveing** [reply]
> Thank you !! My team name is 'sufe-tj'

There is no log generated for your submission. Did you configure your auto-test correctly?

**Reply by luckyyangrun** [reply]
What is the meaning of no log? 
![image](https://user-images.githubusercontent.com/67430592/170404661-c848698d-2323-4c90-ba4f-653f5c89cceb.png)
I have tried several times. The last one is in 2022-05-25 15:58 

> > Thank you !! My team name is 'sufe-tj'
> 
> There is no log generated for your submission. Did you configure your auto-test correctly?

**Reply by QiZhou1997** [reply]
> > > 
> > 
> > 
> > Thank you for quick reply. My team name is ‘Up to 20 characters’. We get the first error message in 2022-05-25 19:57. <img alt="image" width="619" src="https://user-images.githubusercontent.com/26005993/170289194-cf962805-c004-4c89-8cd5-c88f904d8a88.png"> I have tried submitting code several times after that. The last one is in 2022-05-25 22:27. <img alt="image" width="461" src="https://user-images.githubusercontent.com/26005993/170289251-4689493e-0fa1-4f5e-82b3-24f0d01eb87a.png">
> 
> The log displayed "ModuleNotFoundError: No module named 'tqdm'". Did you put 'tqdm' in your pip requirement file?

Thank you! I do not put 'tqdm' in my pip requirement file as I see it has been installed according to https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/preinstalled-pkgs/base_env_installed_packages.md. I will remove tqdm and try submitting again soon.

**Reply by seeingisbelieveing** [reply]
> What is the meaning of no log? ![image](https://user-images.githubusercontent.com/67430592/170404661-c848698d-2323-4c90-ba4f-653f5c89cceb.png) I have tried several times. The last one is in 2022-05-25 15:58
> 
> > > Thank you !! My team name is 'sufe-tj'
> > 
> > 
> > There is no log generated for your submission. Did you configure your auto-test correctly?

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/106231837/170404858-98271fdb-215d-4647-9a6e-ef5b7736159f.png">

There is an error occurred in your submission, please check your code/configuration and try it again. Thanks.

**Reply by luckyyangrun** [reply]
> 
Thank you very much!

**Reply by QiZhou1997** [reply]
> > > 
> > 
> > 
> > Thank you for quick reply. My team name is ‘Up to 20 characters’. We get the first error message in 2022-05-25 19:57. <img alt="image" width="619" src="https://user-images.githubusercontent.com/26005993/170289194-cf962805-c004-4c89-8cd5-c88f904d8a88.png"> I have tried submitting code several times after that. The last one is in 2022-05-25 22:27. <img alt="image" width="461" src="https://user-images.githubusercontent.com/26005993/170289251-4689493e-0fa1-4f5e-82b3-24f0d01eb87a.png">
> 
> The log displayed "ModuleNotFoundError: No module named 'tqdm'". Did you put 'tqdm' in your pip requirement file?

Could you please check the error log again? I still got the error message after I remove the dependence of 'tqdm'. I wonder whether the base environment I use lacks some necessary packages or I submit incorrect code. Thank you.
<img width="860" alt="image" src="https://user-images.githubusercontent.com/26005993/170407128-b2309033-f239-4e7e-b9f6-86db9d151541.png">

**Reply by seeingisbelieveing** [reply]
> > > > 
> > > 
> > > 
> > > Thank you for quick reply. My team name is ‘Up to 20 characters’. We get the first error message in 2022-05-25 19:57. <img alt="image" width="619" src="https://user-images.githubusercontent.com/26005993/170289194-cf962805-c004-4c89-8cd5-c88f904d8a88.png"> I have tried submitting code several times after that. The last one is in 2022-05-25 22:27. <img alt="image" width="461" src="https://user-images.githubusercontent.com/26005993/170289251-4689493e-0fa1-4f5e-82b3-24f0d01eb87a.png">
> > 
> > 
> > The log displayed "ModuleNotFoundError: No module named 'tqdm'". Did you put 'tqdm' in your pip requirement file?
> 
> Could you please check the error log again? I still got the error message after I remove the dependence of 'tqdm'. I wonder whether the base environment I use lacks some necessary packages or I submit incorrect code. Thank you. <img alt="image" width="860" src="https://user-images.githubusercontent.com/26005993/170407128-b2309033-f239-4e7e-b9f6-86db9d151541.png">

Just checked, the running environment is OK. Please check your code/configuration and submit again.

**Reply by caohuixia** [reply]
Dear organizer, I also get stuck with a similar issue.What is the meaning of 'Error Occurred    2022-05-30 20:55', no log? 
My team name is 'learningTeam'.

**Reply by ramber1836** [reply]
An error msg should be returned. Place your mouse pointer on the 'Error Occurred', and the detailed message will show up.

**Reply by caohuixia** [reply]
it's return '异常'！

**Reply by ramber1836** [reply]
Please resubmit and try it again to see whether the detailed msg is returned. 
If not, let us know and we will check it for you. Tips: pls provide your team name and submission time.

**Reply by seeingisbelieveing** [reply]
> it's return '异常'！

Just checked your submission, the arguments that you provide for eval_main.py is unrecognized. Please correct it and submit it again.

**Reply by caohuixia** [reply]
Yesterday，I fix the prepare.py and submit it again, but also get stuck with a similar issue,always return 'Error Occurred  异常'！Please help me check the submission! Thank you !

**Reply by ramber1836** [reply]
> Yesterday，I fix the prepare.py and submit it again, but also get stuck with a similar issue,always return 'Error Occurred 异常'！Please help me check the submission! Thank you !

No log is available.
If the evaluation is terminated by the system due to some reasons, like an out-of-time error, no log is available for us.
We are working on it to avoid the issue that the evaluation is terminated without generating any logs.

**Reply by caohuixia** [reply]
Just  I submit again! Then return '[OOMKilled] job failed because of Insufficient memory' ,so what should I do?

**Reply by ramber1836** [reply]
The memory limit is 16GB in the online test.
Please check your evaluation memory usage in your local machine first.

**Comment by ramber1836** [comment, marked as answer]
The tqdm pkg has already been added to the pytorch environment.
And more detailed error messages will be displayed for your convenience. 
Thanks.

**Reply by jlo404** [reply]
Dear all, I installed all pkgs without any problems but tqdm Apparently, version 0.46.0 does not exist, as you can see in the image below. 
![errorPaqueteTQDM](https://user-images.githubusercontent.com/81811961/171949556-eb1c494b-d781-47ed-8d2a-f9634abf400e.png)

Thanks in advance.
Best regards.

**Reply by ramber1836** [reply]
I don't think it is necessary to specify the version of tqdm in your local machine.
